src/pages/Register.js

```diff
@@ -53,7 +53,6 @@ export const Register = () => {
       navigate('/', { replace: true })
     } catch (e) {
       console.error(e.message)
-      toast.error(e.message)
```

Comment:
The toast should should not be removed from the page in this way. We should display the error message associated with the sign-up so that the user can understand what is the issue.

Reply:
Actually, the error message is already been indicated on the sign-up page, if you follow the instructions I have given for replicating the bug, you'll find out there's one error indication of `Email exist` and also a toast pops up with `data is undefined` text, so I only removed the toast not the actual error message getting displayed.

Reply:
Got it